playbooks/python.yml

```diff
@@ -0,0 +1,10 @@
+---
+# Ensure python is set up and accessible on the server before starting (ansible requires python2.7)
+- name: python
+  hosts: ofn_servers
+  gather_facts: no
+  become: yes
+  roles:
+    - role: python
+      become: yes
+      tags: setup
```

Comment:
Is a separate playbook really needed?

Reply:
We could merge it with `/playbooks/default_user` into a single `/playboks/setup` that does both jobs. Would that be better?

Reply:
After investigating and testing this, it looks like we have to do it this way. 

- We can't use the `apt` module or any other modules before we have run these tasks, or we hit the fatal error.
- We have to use the `gather_facts: no` option or we hit the fatal error. 
- We can only set `gather_facts: no` on a playbook, and all our other playbooks need this option to be on for modules to work, so we need a separate playbook and we can't mix them together.

![screenshot from 2018-07-16 15-01-01](https://user-images.githubusercontent.com/9029026/42762931-872c3b12-8909-11e8-81ba-3ba768cd5ddc.png)